src/bitcoin-cli.cpp

```diff
@@ -867,6 +874,89 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+    std::string RESET = "";
+    std::string GREEN = "";
+    std::string BLUE = "";
+    std::string YELLOW = "";
+    std::string MAGENTA = "";
+    std::string CYAN = "";
```

Comment:
style nit, std::string is default-initialized to `""`
```diff
-    std::string RESET = "";
-    std::string GREEN = "";
-    std::string BLUE = "";
-    std::string YELLOW = "";
-    std::string MAGENTA = "";
-    std::string CYAN = "";
+    std::string RESET, GREEN, BLUE, YELLOW, MAGENTA, CYAN;
```

Reply:
Agreed that is much cleaner 👍 . Updated to this in 7f253cf